virtual-users/build.gradle.kts

```diff
@@ -9,8 +9,7 @@ plugins {
 }
 
 dependencies {
-    compile(fileTree(mapOf("dir" to "lib", "include" to "*.jar")))
-    compile("com.atlassian.performance.tools:jira-actions:[3.0.0,4.0.0)")
+    compile("com.atlassian.performance.tools:jira-actions:[3.6.0,4.0.0)")
```

Comment:
The only bit that worries me is that we didn't port this part: https://github.com/kema2000/jira-actions/commit/72eefabbd15308fbf8c701693dba7f5079b7a564

We should be on the lookout for inflated error counts of `Create Issue` and `Edit Issue` actions. If they're higher now than before, we gotta port that.

Reply:
Do you want me to look into this or just raise a new issue to review later?

Reply:
This was particularly to address an issue during testing against XL Postgresql. The "resolution" filed is configured as mandatory. So it's not an issue for all dataset, I am not 100% sure if it will occur for XL Mysql or not. But we will find out during re-run Mysql XL (quick-240).

Reply:
Let's get CI rolling and then confirm this on CI. QUICK-240 is still running on snapshots.